lib/extended-css.js

```diff
@@ -176,21 +202,26 @@ var ExtendedCss = function (styleSheet) { // jshint ignore:line
             return;
         }
         domChanged = true;
-        setTimeout(handleDomChange, 10);
+
+        if (window.requestAnimationFrame && DomObserver.mutationObserverSupport) {
```

Comment:
Why is requestAnimationFrame invoked only if mutationObserverSupport is supported